code/game/objects/structures/morgue.dm

```diff
@@ -266,17 +266,13 @@ GLOBAL_LIST_EMPTY(crematoriums)
 		update_icon()
 
 		for(var/mob/living/M in conts)
-			if (M.stat != DEAD)
+			if (M.stat == DEAD	|| M.name == "Punishing bird")	//Only kill dead people and punishing bird
```

Comment:
Why are you checking for name?
Also, no need to even add this, punishing bird is being updated already.

Reply:
Its for old time's sake, honestly.